src/libPMacc/include/identifier/alias.hpp

```diff
@@ -59,7 +59,7 @@ identifier(pmacc_isAlias);
 
 /** create an alias
  *
- * an alias is a unspecialized type of a identifier or a value_identifier
+ * an alias is a unspecialized type of an identifier or a value_identifier
```

Comment:
thanks :)

but pls fix typos next time in a separate pull request, such changes just blow up your pull and create merge conflicts for you during rebase and for us during `cherry-pick`.